src/components/DetailsPage.js

```diff
@@ -79,6 +81,37 @@ const DetailsPage = () => {
     type: "",
   });
 
+  const { id } = useParams();
+  const [pet, setPet] = useState([]);
+  const [loading, setLoading] = useState(false);
+  const fetchPetById = async () => {
+    setLoading(true);
+    try {
+      const response = await fetch(
+        `https://pets-v2.dev-apis.com/pets?id=${id}`
+      );
+      const data = await response.json();
+      setPet(data.pets);
+      setLoading(false);
+      console.log(pet);
```

Comment:
remove the console.log. And all other console.log you don't need anymore

Reply:
Okay